src/rules/unit-no-unknown/__test__/index.js

```diff
@@ -0,0 +1,256 @@
+import testRule from "../../../testUtils/testRule"
+
+import rule, { ruleName, messages } from ".."
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
```

Comment:
Let's replace all these `undefined`s with `true`, since that's more aligned with real use-cases & easier to read.